core/jazz_deployments/config/dev-config.json

```diff
@@ -1,20 +1,14 @@
 {
-    "DEPLOYMENT_TABLE": "{conf_stack_prefix}_Deployments_Dev",
+    "DEPLOYMENT_TABLE": "{conf_stack_prefix}_Deployments_Prod",
     "DEPLOYMENT_CREATION_REQUIRED_FIELDS": [
         "service_id",
         "service",
         "domain",
         "environment_logical_id",
-        "provider_build_url",
-        "provider_build_id",
-        "scm_commit_hash",
-        "scm_url",
-        "scm_branch",
-        "status",
-        "request_id"
```

Comment:
i think you should keep request_id in this list.

Reply:
it was throwing error. I already point out this thing to you.